torch/_torch_docs.py

```diff
@@ -2198,16 +2198,14 @@ def merge_dicts(*dicts):
     tensor([(0.0000+1.0000j), (-1.4142-1.4142j)], dtype=torch.complex128)
 """)
 
-add_docstr(torch.conj,
+add_docstr(torch.conj_physical,
            r"""
-conj(input, *, out=None) -> Tensor
+conj_physical(input, *, out=None) -> Tensor
 
-Computes the element-wise conjugate of the given :attr:`input` tensor. If :attr:`input` has a non-complex dtype,
-this function just returns :attr:`input`.
+Computes the element-wise conjugate of the given :attr:`input` tensor. If :attr:`input` has a non-complex dtype, this function just returns :attr:`input`.
 
-.. warning:: In the future, :func:`torch.conj` may return a non-writeable view for an :attr:`input` of
-             non-complex dtype. It's recommended that programs not modify the tensor returned by :func:`torch.conj`
-             when :attr:`input` is of non-complex dtype to be compatible with this change.
+.. note::
+   This performs the conjuate operation regardless of the fact conjugate bit is set or not.
```

Comment:
"conjuate" -> "conjugate"

The idea of a "conjugate bit" needs to be introduced before this note. Maybe it would be helpful to describe how this function is different than conj()?

If a user does `t.conj().conj_physical()` do they get a copy of the original tensor back?